flexmeasures/ui/templates/crud/account.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}{% block title %} {% endblock %} {% block divs %} {% if account %}
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-sm-8">
+            <div class="account-data-table card">
+                <h2>Account overview</h2>
+                <small>Account: {{ account["name"] }}</small>
+                <table class="table table-striped table-responsive">
+                    <tbody>
+                        <tr>
+                            <td>
+                                ID
+                            </td>
+                            <td>
+                                {{ account.id }}
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                Account Roles
+                            </td>
+                            <td>
+                                {{ account.account_roles }}
```

Comment:
Would it make sense to comma separate this list here, too?
```
{{ account.account_roles | join(", ") }}
```

Reply:
Implemented